.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please start by describing concisely the nature of the bug or the feature
+request.
+
+In case of a bug report, please describe the steps how the problem may be
+reproduced. Does the problem occur on production servers? Does it work on the
+development server? Can you reproduce it locally on the `master` branch?
+
+In case of a feature request, please provide the motivation and the concrete use
+case which your request is seeking to address.
+
+Finally, before opening the issue, please select appropriate labels to indicate
+the topic and the complexity of the task. Please select the concrete milestone
+if applicable.
```

Comment:
Can non-members select labels? I don't think so. Maybe we shouldn't confuse people, so I would say, "If you are a member of the `cernopendata` team, please make sure you select appropriate labels…"